src/pivot-table/components/cells/utils/should-render-column-adjuster.ts

```diff
@@ -0,0 +1,5 @@
+import type { Cell } from "../../../../types/types";
+
+export default function shouldRenderColumnAdjuster(cell: Cell, isActive: boolean) {
```

Comment:
went back and forth on this. I'd prefer if ColumnAdjuster took care of its own render condition. The thing is if we add <ColumnAdjuster> to all dimension and pseudo dimension cells, there is a useEffect that runs on all of them, so I went with just not rendering it at all. The logic for rendering it will become more complex when it is added to the header grid